oak_functions/loader/src/server.rs

```diff
@@ -149,3 +296,63 @@ pub async fn create_and_start_server(
     );
     Ok(())
 }
+
+/// A resolver function, mapping Oak host function names to an index and a type signature.
+fn oak_resolve_func(
```

Comment:
Why does this function have an `oak_` prefix?
Should it just be a `resolve_function` ?

Reply:
I think it is good to have different names. Now to be more accurate, I changed it to `oak_functions_resolve_func`, since the module name is `oak_functions` :D